tests/rptest/scale_tests/many_partitions_test.py

```diff
@@ -762,6 +762,14 @@ def test_omb(self):
             replication_factor=3,
             mib_per_partition=DEFAULT_MIB_PER_PARTITION,
             topic_partitions_per_shard=DEFAULT_PARTITIONS_PER_SHARD)
+
+        self.redpanda.add_extra_rp_conf({
+            'topic_partitions_per_shard':
+            DEFAULT_PARTITIONS_PER_SHARD,
+            'topic_memory_per_partition':
+            int(DEFAULT_MIB_PER_PARTITION * 1024 * 1024),
```

Comment:
Which of the two settings here is the operative one for the CI runs in question?

Reply:
You have commented on the (backport) commit which sets the limits for the OMB test. This was made to set the limits at all. Otherwise it would just use the all the memory of the machine so wouldn't do what the test intended at all (taget partition/memory density).

But now yes it would also be needed in any case as we are changing `topic_memory_per_partition` for all the tests (not just the OMB one).

Reply:
I'm a bit lost at this point, but I think this is all good regardless.